### ISSUE TITLE
CaMN pretrained-model or training parameters needed

### ISSUE THREAD

**Issue by SanGibb** [PantoMatrix/PantoMatrix#14]
Nice Work!
I'm having trubble while using CaMN as baselie.
I trained CaMN on your BEAT dataset with parameters shown in this scripts. I don't know whether because I close emotion modality and sem modality or other reasons, I got same FGD as shown in your paper, but the result looks not as good as your demo.
Will you realease pretrained model or at least provide training config? Thanks a lot.

**Comment by H-Liu1997** [comment]
Hi, thank you very much for using BEAT dataset. 

- I just noticed one mistake for the code version, the nn.GRU in camn.py should be replaced by nn.LSTM. Are your results based on LSTM version?  

- I checked the training config, I adopt adam, lr 5e-4, batchsize 256, no adv epochs 40, and adv weight 0.5 for training (smaller or remove adv loss for speakers with higher motion diversity will get smoother results).  

- I will relsease the pretrained model by the end of this year due to the application of approvement in the company. 

Thanks!

**Comment by H-Liu1997** [comment]
hi @SanGibb 

sorry for the late reply, you may download the ckpts from [this link](https://drive.google.com/drive/folders/1E5lMp0Kc7SV81fJpTiQcOJ3g6Umz6Vtn?usp=sharing),

I also updated the training config and readme for details.

Thanks!
